config/projects/fuzzing.yml

```
@@ -31,7 +31,7 @@ fuzzing:
       owner: jkratzer@mozilla.com
       emailOnError: false
       imageset: docker-worker-2022
-      cloud: gcp
+      cloud: aws
```

Now you should add this security group since it's moving over to AWS.

```suggestion
      cloud: aws
      securityGroup: docker-worker
```